src/merge.js

```diff
@@ -280,9 +268,8 @@ class Merge {
       }
       markSide(side, doc, folder)
       markSide(side, was, was)
-      if (doc.creationDate == null) { doc.creationDate = was.creationDate }
       if (doc.tags == null) { doc.tags = was.tags || [] }
-      delete doc.toBeTrashed
+      delete doc.trashed
```

Comment:
Old name was `toBeTrashed` because I believed (maybe I was wrong) that the operation was "in progress", and the flag was supposed not to be present anymore in the metadata of the trashed content. But let's discuss this later. I'll look at the code below for better understanding.